response_operations_ui/templates/collection_exercise/ce-sample-ci-section.html

```diff
@@ -1,66 +1,88 @@
+                    "variants": ['compact', 'responsive'],
+                    "id": "sample-table",
+                    "caption": 'Loaded sample summary',
+                    "hideCaption": true,
+                    "ths": [
+                        { 
+                            "value": "Sample & Collection instruments (CI)",
+                            "thClasses": "ons-u-fs-r--b"
+                        },
+                        {
+                            "value": "",
+                            "thClasses": "ons-u-ta"
+                        },
+                        {
+                            "value": "Uploaded " + '<em>' + sample.ingestDateTime[:-8] + '</em>' + sample.ingestDateTime[-8:],
+                            "thClasses": "ons-u-fs-r",
+                            "numeric": "true"
+                        },
+                    ],
+                    
+                    "trs": [
+                        {
+                            "tds": [
+                                {
+                                    "value": "Total businesses",
+                                    "tdClasses": "ons-u-ta"
+                                },
+                                {
+                                    "value": sample.totalSampleUnits,
+                                    "name": "total-instruments",
+                                    "tdClasses": "ons-u-ta-right"
+                                },
+                                {
+                                    "value": ' ',
+                                    "name": "upload-sample-time",
+                                }
+                            ]
+                        },
+                        {
+                            "tds": [
+                                {
+                                    "value": "Collection instruments",
+                                    "tdClasses": "ons-u-ta"
+                                },
+                                {
+                                    "value": sample.expectedCollectionInstruments,
+                                    "name": "total-instruments",
+                                    "tdClasses": "ons-u-ta-right"
+                                },
+                                {
+                                    "value": ' ',
+                                    "name": "upload-sample-time",
+                                }
+                            ]
+                        }
+                    ]
+                })
+            }}
+        {% elif 'INIT' == sample.state %}
+            <div class="ons-u-mb-xl">
+                <div class="ons-u-fs-r">
+                    Loading&hellip; <span class="ons-u-ml-xs"><a href="/surveys/{{survey.shortName}}/{{ce.exerciseRef}}/view-sample-ci?show_msg=true">Refresh to see progress</a></span>
+                </div>
+            </div>
+        {% elif 'FAILED' == sample.state %}
```

Comment:
How many sample states do we have? Should one not be default, i.e else?

Reply:
There are 4, it seems that if the state is 'COMPLETE' nothing is being rendered. For now I've set the else condition to if the state is 'ACTIVE' or 'COMPLETE', I will check with Gareth what each state should look like and add that

Reply:
The database does constrain it to the 4 states but just in case the else at the moment will show just the header I will check with Gareth what we want it to say